Lib/fontmake/font_project.py

```diff
@@ -541,6 +540,12 @@ def save_otfs(
                 if "GSUB" in gsub_src:
                     font["GSUB"] = gsub_src["GSUB"]
 
+            # Read autohinting parameters from ufo lib if present
+            if "com.schriftgestaltung.customParameter.InstanceDescriptorAsGSInstance.TTFAutohint options" in ufo.lib and autohint is None:
```

Comment:
Please put this long string as a constant at the top of the file, similar to `KEEP_GLYPHS_NEW_KEY`